example/20-seedmanager-admissionplugin-configuration.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: apiserver.k8s.io/v1alpha1
```

Comment:
Rename file to `20-admissionconfiguration-seedmanager.yaml`

Reply:
Also, please make it possible to provide this configuration via the [Gardener Helm chart](https://github.com/gardener/gardener/tree/master/charts/gardener).